LayoutTests/http/wpt/html/links/redirect-301-with-fragments-test-network-cache.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <script src='/resources/testharness.js'></script>
+    <script src='/resources/testharnessreport.js'></script>
+    <title>Ensure fragments are correctly transferred on redirects. Disable the memory cache to ensure the network cache is being tested.</title>
+</head>
+<body>
+    <script>
+        /* Clear the memory cache and disable it */
+        internals.clearMemoryCache();
+        internals.pruneMemoryCacheToSize(0);
```

Comment:
I would think this is equivalent to clearing. How does this disable the memory cache?

Reply:
As I thought, this test doesn't cover the disk cache fix. I have verified by running the test locally without this part of the fix. Please address.